### PR TITLE
fix(targets): linux/ia32 official support was removed in Electron 19

### LINKED ISSUE #1447: -all should not attempt to build Linux/ia32 since it fails to find required files
### Preflight Checklist

* [ X] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/main/CONTRIBUTING.md) for this project.
* [X ] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* X[ ] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
Electron Packager 17.1.1
Node v18.9.0
Host Operating system: win32 10.0.22621 (x64)
* **Electron Version:**
v21.3.0
* **Operating System:**
Host Operating system: win32 10.0.22621 (x64)* 
**Last Known Working Electron Packager version:**:
  * 

### Expected Behavior
Since Linux/ia32 is deprecated for the version of Electron electron-package should not attempt to build it when --all is specified

### Actual Behavior
electron-packager fails with 404 error

C:\Development\YADAMU>npx electron-packager ./src yadamu --all  --out=c:\dist --overwrite
Official Linux/ia32 support is deprecated.
For more info: https://electronjs.org/blog/linux-32bit-support
Packaging app for platform darwin universal using electron v21.3.0 - Building x64 and arm64 slices now
Packaging app for platform mas universal using electron v21.3.0 - Building x64 and arm64 slices now
Packaging app for platform win32 x64 using electron v21.3.0
Response code 404 (Not Found) for https://github.com/electron/electron/releases/download/v21.3.0/electron-v21.3.0-linux-ia32.zip

### PR DESCRIPTION
**Pull request by malept**
* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/main/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] The changes have sufficient test coverage (if applicable).
* [ ] The testsuite passes successfully on my local machine (if applicable).

**Summarize your changes:**

Compare the assets of [19.0.0-alpha.5](https://github.com/electron/electron/releases/tag/v19.0.0-alpha.5) with [19.0.0-beta.1](https://github.com/electron/electron/releases/tag/v19.0.0-beta.1).

Fixes #1447

### CODE REVIEW

File: src/targets.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const buildVersions = {
   },
   linux: {
     arm64: '>= 1.8.0',
+    ia32: '<19.0.0-beta.1',
```
Comment (erickzhao) [inline src/targets.js:23]: Was this an alpha/beta cutoff change or an 18/19 cutoff change?

```suggestion
    ia32: '<19.0.0-alpha.1',
```

Comment (malept) [reply src/targets.js:23]: It wasn't the latter. From the PR summary:

> Compare the assets of [19.0.0-alpha.5](https://github.com/electron/electron/releases/tag/v19.0.0-alpha.5) with [19.0.0-beta.1](https://github.com/electron/electron/releases/tag/v19.0.0-beta.1).

I'd have to look at either PRs or Slack context to see what the issue was (if I had to guess, ia32 didn't actually work in alpha, but :shrug:)

Comment (erickzhao) [reply src/targets.js:23]: From some internal maintainer discussion, we technically we publicly announced that we were going to drop support for Linux in Electron 4 but then never removed it from CI: https://www.electronjs.org/blog/linux-32bit-support

Comment (erickzhao) [reply src/targets.js:23]: I think I'm good with either option, though.

**Review by erickzhao — APPROVED** [review]

**Pull request merged**